operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -1533,17 +1533,15 @@ local utils = import 'mixin-utils/utils.libsonnet';
       $.row($.capitalize(rowTitlePrefix + 'query-frontend'))
       .addPanel(
         $.timeseriesPanel('Requests / sec') +
-        $.qpsPanel('cortex_request_duration_seconds_count{%s, route=~"%s"}' % [$.jobMatcher(queryFrontendJobName), queryRoutesRegex])
+        $.qpsPanelNativeHistogram('cortex_request_duration_seconds', utils.toPrometheusSelectorNaked($.jobSelector(queryFrontendJobName) + [utils.selector.re('route', queryRoutesRegex)]))
```

Comment:
I don't know if it will be needed, but maybe `cortex_request_duration_seconds` can be replaced with a `metrics` variable.